django/db/models/fields/related_descriptors.py

```diff
@@ -798,6 +798,81 @@ def _get_set_deprecation_msg_params(self):
             self.rel.get_accessor_name() if self.reverse else self.field.name,
         )
 
+    def add_relations(self, relations, *, assert_no_collisions=False):
+        """
+        Creates many (M1, M2) relations with O(1) database queries.
+
+        Any requested relations that already exist will be left alone.
```

Comment:
"Any requested relations that already exist will not be altered." is perhaps better wording?

Reply:
Will do.